tests/integration/test_emergency_contacts.py

```diff
@@ -96,7 +96,12 @@ def test_emergency_contacts_POST(client, auth_headers):
     assert is_valid(
         response, 400
     )  # BAD REQUEST - {'name': 'This Field Cannot Be Blank.'}
-    assert response.json == {"message": {"name": ["Missing data for required field."]}}
+    assert response.json == {
+        "message": {
+            "name": ["Missing data for required field."],
+            "contact_numbers": ["Missing data for required field."],
+        }
```

Comment:
I'd say we can delete this assert. It's retesting what the schema already tests. And testing the response message is testing the Marshmallow library. We can create an issue to refactor these tests. I think all of the tests except the one testing the update action needs to be refactored, because the update action hasn't been refactored yet in the resource.